core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -1641,7 +1641,8 @@ KOKKOS_INLINE_FUNCTION
   for (iType i = loop_boundaries.start; i < loop_boundaries.end; i++) {
     ValueType tmp = ValueType();
```

Comment:
This should likely also be replaced by `reducer.init(...)`.

Reply:
Yes. Its a WIP. Sorry forgot to mark that label. Currently that loop is not encountered though.